File: JavaStudy/src/main/java/basicStudy/WildcardCharacter/Test1.java
package basicStudy.WildcardCharacter;

import java.util.ArrayList;
import java.util.List;

/**
 * @ClassName Test1
 * @Description
 * @Author SDY
 * @Date 2024/3/14 22:04
 **/
public class Test1 {
    public static void main(String[] args) {
        List<?> list = null;
        list = new ArrayList<String>();
        list = new ArrayList<Double>();
        //list.add(3);
        list.add(null);


    }
}

File: FlinkStudy/src/main/scala/Test/Test1.scala
package Test

import org.apache.flink.api.scala.createTypeInformation
import org.apache.flink.streaming.api.scala.{DataStream, StreamExecutionEnvironment}

import java.util.UUID

/**
 * @ClassName: Test1
 * @Auther: SDY
 * @Description:
 * @Date: 2024/4/7 22:29
 * */
object Test1 {
  def main(args: Array[String]): Unit = {
    val id = UUID.randomUUID().toString
    print(id)
  }

}
